packages/amplify-category-geo/src/__tests__/service-stacks/__snapshots__/placeIndexStack.test.ts.snap

```diff
@@ -508,10 +496,10 @@ exports.handler = async function (event, context) {
     if (event.RequestType == 'Update') {
       const params = {
         IndexName: event.ResourceProperties.indexName,
-        PricingPlan: event.ResourceProperties.pricingPlan,
         DataSourceConfiguration: {
           IntendedUse: event.ResourceProperties.dataSourceIntendedUse,
         },
+        PricingPlan: 'RequestBasedUsage'
```

Comment:
nit: consider pulling this value out into a const somewhere and reusing in all these locations

Reply:
This is in the snapshots where it's just a string as expected. I made changes to the lambda handlers and made the pricing plan a constant there.